Tests/RouterTests.swift

```diff
@@ -150,6 +156,26 @@ class RouterTests: QuickSpec {
                 expect(usersCommentsInfo?.queryParameters).toEventually(equal([NSURLQueryItem(name: "page", value: "2"), NSURLQueryItem(name: "author", value: "hector")]))
                 expect(usersCommentsResponseURL?.absoluteString).toEventually(equal("http://www.test.com/users/1/comments/2?page=2&author=hector"))
             }
+            
+            context("when the Router has latency") {
+                it("should delay the mocked response") {
+                    var responseData: NSData? = nil
+                    router.latency = 2.0
+                    router.get("/users/:id") { request in
+                        return ["test": "value"]
+                    }
+                    
+                    NSURLSession.sharedSession().dataTaskWithURL(NSURL(string: "http://www.test.com/users/1")!) { (data, response, _) in
+                        responseData = data
+                        }.resume()
+                    
+                    
+                    let startTime = CFAbsoluteTimeGetCurrent()
+                    expect(responseData).toNotEventually(beNil(), timeout: 3)
+                    let endTime = CFAbsoluteTimeGetCurrent()
+                    expect(endTime - startTime) >= 2.0
+                }
+            }
```

Comment:
Probably good idea to test that adding latency to a router doesn't affect other router's latency.